### PR TITLE
Strong name Microsoft.AspNet.JsonPatch.

### PR DESCRIPTION
**Pull request by cesarbs**
https://github.com/aspnet/dnx/issues/2615

cc @davidfowl @muratg @Eilon @moozzyk @pranavkm @Tratcher

### CODE REVIEW

File: src/Microsoft.AspNet.JsonPatch/project.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   "version": "1.0.0-*",
+  "compilationOptions": {
+    "keyFile": "../../tools/Key.snk"
+  },
```
Comment (Eilon) [inline src/Microsoft.AspNet.JsonPatch/project.json]: `warningsAsErrors` :wink:

**Comment by cesarbs** [comment]
@Eilon Added warningsAsErrors.

**Comment by cesarbs** [comment]
Ping.

**Comment by pranavkm** [comment]
:shipit:

**Pull request merged**